designs/2022-community-eslint-org/README.md

```diff
@@ -71,10 +71,10 @@ the updates would just resume).
 
 Packages that are living under the new community org would be allowed to (keep)
 accept(ing) sponsorship for the maintainers of that package.  
-If the community core team at some point decides to setup its own sponsorship
-collective (like the core ESLint org has its
-[Open Collective](https://opencollective.com/eslint)), packages could also
-include this collective & the community org in their `FUNDING.yml`.
+The community core team will also setup its own sponsorship collective (like the
```

Comment:
I feel like this would be confusing. If we are going to pay the org maintainers from the core fund, why would there be another collective too?

I feel like this would be confusing to everyone.

I have no objections to projects having their own collectives to donate to, it just seems like if eslint-community is part of the larger ESLint project, then the funding should all come from the core fund.

Reply:
Yep, that's the same thing we do for [webpack](https://github.com/webpack/) and [webpack-contrib ](https://github.com/webpack-contrib)

Reply:
@MichaelDeBoey I think this is the last part that needs resolution, what do you think?

Reply:
Oh then I think I misunderstood.

I though we would have the following flow

ESLint collective => ESLint community collective => community core team

But you're saying it would be

ESLint collective => community core team

Is that correct?

Reply:
Yes, that’s correct.